src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ImportService.cs

```diff
@@ -62,7 +62,7 @@ public class ImportService : IImportService
             aQueue.CreateIfNotExists();
             // TODO - EES-1250
             var numRows = isZip ? 0 : FileStorageUtils.CalculateNumberOfRows(dataFile.OpenReadStream());
-            var message = BuildMessage(dataFileName, metaFileName, releaseId, isZip ? dataFile.FileName.ToLower() : "");
+            var message = BuildMessage(dataFileName, metaFileName, releaseId, 1, 1, isZip ? dataFile.FileName.ToLower() : "");
```

Comment:
Named parameters would be a good idea here as you have two identical values sat next to each other. At a glance, you can't really figure out what parameters these actually line up with.